core/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/DefaultHttpJsonRequest.java

```diff
@@ -118,6 +121,16 @@ public HttpJsonRequest addQueryParam(@NotNull String name, @NotNull Object value
     return this;
   }
 
+  public HttpJsonRequest addRequestHeader(@NotNull String name, @NotNull String value) {
```

Comment:
Consider renaming to `addHeader`. We can not add response header to a request.

Reply:
ok, lemme do it